app/locales/pt-BR.json

```diff
@@ -0,0 +1,118 @@
+    "event_history_title": "Histórico de exposição",
+    "export_para_1": "Se o seu teste para o COVID-19 deu positivo, por favor faça sua parte compartilhando o seu histórico de localização para as autoridades de saúde locais.",
+    "export_para_2": "A sua localização é compartilhada como uma lista simples de horários e locais, sem nenhuma informação além disso.",
+    "home_at_risk_header": "Você pode ter sido exposto",
+    "home_at_risk_subsubtext": "Isso não significa que você está infectado.",
+    "home_at_risk_subtext": "Baseado em seu histórico de localização, é possível que você esteve em contato ou próximo a alguém diagnosticado com COVID-19.",
+    "home_enable_location": "Ativar dados de localização",
+    "home_mayo_link_heading": "Mais informações sobre o COVID-19",
+    "home_mayo_link_label": "de Mayo Clinic",
+    "home_no_contact_header": "Nenhum contato conhecido",
+    "home_no_contact_subtext": "Com base nos dados disponíveis, você não esteve perto de ninguém que testou positivo para o COVID-19.",
+    "home_unknown_header": "Desconhecido",
+    "home_unknown_subtext": "Não podemos dizer se você está em risco, a menos que você permita o aplicativo acessar sua localização.",
+    "home_setting_off_header": "Desconhecido",
+    "home_setting_off_subtext": "Não podemos dizer se você está em risco, a menos que você ative seu histórico de localização na tela de configurações.",
+    "import_step_1": "1. Realize login na sua conta do Google e baixe o seu histórico de localização",
+    "import_step_2": "2. Depois que você baixar, abra esta tela novamente. Os dados serão importados automaticamente.",
+    "import_title": "Importar localização",
+    "latest_news": "Últimas notícias",
+    "launch_done_header": "Tudo pronto",
+    "launch_done_subheader": "Você está pronto para seguir. Lembre-se, você sempre pode alterar suas preferências depois. ",
+    "launch_enable_location": "Ativar localização",
+    "launch_enable_notif": "Ativar notificações",
+    "launch_finish_set_up": "Concluir configurações",
+    "launch_get_started": "Iniciar",
+    "launch_location_access": "Acesso à localização",
+    "launch_location_header": "Para lembrar onde você deve ir, seu celular precisa ter sua localização",
+    "launch_location_subheader": "Não se preocupe, as informações nunca saem do seu dispositivo, a menos que você decida explicitamente que deseja compartilhá-las.",
+    "launch_next": "Próximo",
+    "launch_notif_header": "Você receberá uma notificação se cruzar o caminho com uma pessoa infectada.",
+    "launch_notif_subheader": "Não iremos incomodá-lo, exceto para compartilhar atualizações sobre seus potenciais riscos de exposição.",
+    "launch_notification_access": "Permitir notificações",
+    "launch_screen1_header": "O caminho de volta ao normal começa aqui.",
+    "launch_screen2_header": "Seja notificado se você encontrar alguém diagnosticado para COVID-19.",
```

Comment:
This is the English line:
"Get notified if you cross paths with someone later diagnosed for COVID-19."
Should the above translation be on the lines of:
Seja notificado se você cruzar o caminho de alguém que tenha recebido diagnostico do COVID-19 mais tarde .?